covasim/webapp/cova_app.py

```diff
@@ -207,8 +160,71 @@ def get_gantt(intervention_pars=None, intervention_config=None):
 
     return response
 
+
+#%% Define the core API
+
+def parse_interventions(int_pars):
```

Comment:
The interventions should mostly all have the same API so my concern with separate functions is that it would introduce a lot of duplication. We could/should probably even collapse the if statement a bit, pull parameters out to the front, etc.

Reply:
Duplication isn't the enemy. It can give you more flexibility in the future and make it more comprehensible to newcomers.
Given that the number and quantity of interventions is still in flux (assuming changes post-feedback from stakeholders), it would be premature to solidify the intervention api too much.

Reply:
It's definitely a balance, and if we end up with something significantly more complicated, we might want to refactor. Since I'll probably be the one working the most on this part of the code for the time being, and since this will be easiest the way things currently are, I'd prefer to keep it as-is for now